store/iavl/store.go

```diff
@@ -23,6 +24,7 @@ import (
 
 const (
 	DefaultIAVLCacheSize = 500000
+	CommitQueueBuffer    = 10 // TODO configurable
```

Comment:
```suggestion
	CommitQueueBuffer    = 10 // maximum pending commit versions.
```

I guess we don't actually need to make it configurable?